scripts/release/debian/build.sh

```diff
@@ -21,8 +21,17 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Update APT packages
 apt-get update
 # uuid-dev is used to build _uuid module: https://github.com/python/cpython/pull/3796
-apt-get install -y libssl-dev libffi-dev python3-dev debhelper zlib1g-dev uuid-dev
-apt-get install -y wget
+apt-get install -y libssl-dev libffi-dev python3-dev zlib1g-dev uuid-dev wget
+
+# In Ubuntu 18.04, debhelper 11.1.6 has bug which makes it fail to dpkg-buildpackage. Use backport version instead.
```

Comment:
How does it fail? What is the error message?

Reply:
See https://dev.azure.com/azclitools/public/_build/results?buildId=21215&view=logs&jobId=08788da6-2012-5326-c229-329cadca6b1f&j=08788da6-2012-5326-c229-329cadca6b1f&t=ab47f3d3-16e4-548d-642b-71ea2a3d9689
```
2022-12-19T06:49:15.1225694Z dpkg-deb: building package 'azure-cli' in '../azure-cli_2.43.0-1~bionic_arm64.deb'.
2022-12-19T06:49:15.1282741Z 	install -d debian/.debhelper/scratch-space/build-azure-cli
2022-12-19T06:49:15.1283725Z 	dpkg-deb --build debian/.debhelper/azure-cli/dbgsym-root debian/.debhelper/scratch-space/build-azure-cli
2022-12-19T06:49:15.1306784Z dpkg-deb: building package 'azure-cli-dbgsym' in 'debian/.debhelper/scratch-space/build-azure-cli/azure-cli-dbgsym_2.43.0-1~bionic_arm64.deb'.
2022-12-19T06:49:20.3603124Z dh_builddeb: mv debian/.debhelper/scratch-space/build-azure-cli/azure-cli-dbgsym_2.43.0-1\~bionic_arm64.deb ../azure-cli-dbgsym_2.43.0-1\~bionic_arm64.ddeb: Invalid cross-device link
2022-12-19T06:49:20.3605640Z 	Renaming azure-cli-dbgsym_2.43.0-1~bionic_arm64.deb to azure-cli-dbgsym_2.43.0-1~bionic_arm64.ddeb
2022-12-19T06:49:20.3607058Z 	mv debian/.debhelper/scratch-space/build-azure-cli/azure-cli-dbgsym_2.43.0-1\~bionic_arm64.deb ../azure-cli-dbgsym_2.43.0-1\~bionic_arm64.ddeb
2022-12-19T06:50:31.2456168Z dh_builddeb: Aborting due to earlier error
2022-12-19T06:50:31.2505025Z make: *** [binary] Error 255
2022-12-19T06:50:31.2518914Z debian/rules:8: recipe for target 'binary' failed
2022-12-19T06:50:31.2519828Z dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
2022-12-19T06:50:32.2088117Z ##[error]Bash exited with code '2'.
2022-12-19T06:50:32.2247041Z ##[section]Finishing: Build ubuntu bionic arm64 Package
```